packages/server/graphql/mutations/endTeamPrompt.ts

```diff
@@ -0,0 +1,74 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import EndTeamPromptPayload from '../types/EndTeamPromptPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {GQLContext} from '../graphql'
+import MeetingTeamPrompt from '../../database/types/MeetingTeamPrompt'
+import standardError from '../../utils/standardError'
+
+const endTeamPrompt = {
+  type: GraphQLNonNull(EndTeamPromptPayload),
+  description: `Finish the team prompt meeting`,
+  args: {
+    meetingId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The meeting to end'
+    }
+  },
+  resolve: async (
+    _source: unknown,
+    {meetingId}: {meetingId: string},
+    {authToken, dataLoader, socketId: mutatorId}: GQLContext
+  ) => {
+    const r = await getRethink()
+    const viewerId = getUserId(authToken)
+    const now = new Date()
+    const operationId = dataLoader.share()
+    const subOptions = {mutatorId, operationId}
+
+    // AUTH
+    const meeting = (await r
```

Comment:
+1 I think we should use
```
const meeting = dataLoader.get('newMeetings').load(meetingId) as MeetingTeamPrompt | null
```
here. You just need to make sure after you've updated it in rethink to also update `meeting.endedAt = now`

Reply:
what's the advantage of using a data loader here instead of a raw query?

Reply:
Data will be cached in the dataLoader already when publishing the subscription, so 1 less load from DB.

Reply:
right! fixed.